portfolio/src/main/webapp/index2.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Vincent Stevenson</title>
+        <link rel="stylesheet" href="style2.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <h1>Favorite Games!</h1>
+        <div id="div1">
+            <p>As an avid gamer, I thought that I would make a 
+                list of my favorite games to play and/or have 
+                completed!
+            </p>
+            <ul>
+                <li><a href="#Destiny">Destiny</a></li>
+                <li><a href="#Smite">Smite</a></li>
+                <li><a href="#Spyro:_Reignited_Trilogy">Spyro: Reignited Trilogy</a></li>
+            </ul>
+        </div>
+        </br>
+        </br> 
+        <div id="div2">
```

Comment:
These ids should also be more descriptive. Div2 doesn't indicate much about what is in here, and it becomes confusing to read in the CSS as well